src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/PackageFeeds/InstalledAndTransitivePackageFeed.cs

```diff
@@ -29,16 +29,84 @@ public override async Task<SearchResult<IPackageSearchMetadata>> ContinueSearchA
             cancellationToken.ThrowIfCancellationRequested();
 
             // Remove transitive packages from project references. Those transitive packages don't have Transitive Origins
-            IEnumerable<PackageCollectionItem> transitivePkgsWithOrigins = _transitivePackages
-                .Where(t => t.PackageReferences.All(x => x is ITransitivePackageReferenceContextInfo y && y.TransitiveOrigins.Any()));
+            IEnumerable<PackageCollectionItem> transitivePkgsWithOrigins = GetTransitivePackagesWithOrigins(_transitivePackages);
+
+            // Get the metadata for the latest version of the installed packages and transitive packages
+            IEnumerable<PackageCollectionItem> installedPackagesLatest = GetLatestPackages(_installedPackages);
```

Comment:
Nit: can this be updated to match the other variable name? `latestInstalledPackages`